requirements.txt

```diff
@@ -69,6 +69,7 @@ sortedcontainers==2.4.0
 soupsieve==2.5
 splinter==0.19.0
 SQLAlchemy==2.0.21
+subtitle-parser==1.3.0
```

Comment:
This will also need to be added to the `pyproject.toml` file, which handles dependencies separately.